### ISSUE TITLE
Package manager support?

### ISSUE THREAD

**Issue by MoridinBG** [ernesto-elsaesser/OfflineWebView#5]
Hello, do you plan to add package manager support - Cocoapods, Carthage or SPM?

I have added just the WebArchiver to a proof of concept project and I really like your approach and the code style. I want to add it as a proper external dependency to be able to pull updates or maybe upstream mine.

**Comment by ernesto-elsaesser** [comment]
Hi MoridinBG,

thanks for the nice words :)

My plan is to provide the WebArchiver as an SPM package. However, the code depends on Fuzi (https://github.com/cezheng/Fuzi), which uses `libxml2`, which doesn't play nice with SPM (see https://forums.swift.org/t/referring-to-libxml2-in-swift-package-description/28880).

The good news is that Xcode 11.4 will fix that. So as soon as it's out of beta (probably this month), I will add SPM support. Is SPM an option for you?

I'm not planning to provide CocoaPods or Carthage support for now.

**Comment by ernesto-elsaesser** [comment]
Hi MoridinBG,

I just made the `WebArchiver` available as a standalone Swift package:
https://github.com/ernesto-elsaesser/WebArchiver

As Fuzi didn't release a new version yet, my package still depends on Fuzi via branch name. Because of this, SPM also forces others to depend on the WebArchiver package via branch name (i.e. `master`). As soon as Fuzi releases their Xcode 11.4 fix, I will release a 1.0.1 version that can be depended on via version number.

I already updated the OfflineWebView project to use the `WebArchiver` via Swift Package Manager.

If you decide to use the package, let me know if everything works for you!
